authentication/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "uk.gov.android.authentication"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 29
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
```

Comment:
Does your `buildLogic` project have a plugin for it that relates to setting up java configuration? If there is, I think you're able to configure / use that, then this block can be removed.

However, I don't know whether you would have an 'android library' version of the plugin in there. I believe the wallet repository would have that version as I remember talking it through with @chrisdondig .

Reply:
In jira ticket